### PR TITLE
New Features

### PR DESCRIPTION
**Pull request by IncognitoCoding**
### **Feature Highlights:**

**HTML Conversion**
- The biggest challenge that I have found with push notifications is email messages sent in HTML format. Most notification services through apprise (I use Gotify) only support text. Gotify offers a markdown option, but it does not strip/remove the HTML. My HTML conversion strips all the HTML away and adds formatting to keep the message similar to the HTML output but in text format.
- I created the ability to add the option to convert any HTML message to text per the configuration section in the mailrise.conf file.

**Message Encryption**
- Not many services offer the ability to encrypt, but using mailrise as a primary SMTP relay would allow any message getting sent to get encrypted.
- I use the cryptography library and allow the user to set an encryption password and random salt. I created the ability to add encryption per the configuration section in the mailrise.conf file.

**Message Decryption Website**
- Having encryption is excellent, but having an easy way to decrypt the message through a website is even nicer.
- I use the same encryption password and random salt from the mailrise.conf file to launch a website to decrypt the message.
- The decryption page has a fully customizable HTML page that can be edited by any user that wishes to add their flare to the page.

### CODE REVIEW

**Comment by YoRyan** [comment]
Hi,

I appreciate the effort you put into your fork, and this PR, but I'm just not sure if I can accept it, because of how heavily it depends on your personal ictoolkit library. I aim to minimize the number of dependencies in my projects, and when I do take them on, I try to stick to well-supported, battle-tested ones. For Mailrise that presently includes aiosmtpd, PyYAML, and, obviously, Apprise. I mean no offense, but I think it's fair to say that ictoolkit has not yet reached that aforementioned level of maturity.

But conceptually, I would be quite interested in an HTML to text converter; as you correctly note, very few notification targets support HTML, and Apprise dose not supply a converter for those that don't. As for encryption, I personally think this is too niche of a use case for mainline Mailrise. Considering the average threat model, most users should be satisfied with a self-hosted notification server that supports TLS, plus notification hiding on their smartphones.

In short, I would be happy to accept code directly integrated into Mailrise that better handles HTML mail, but I think it would be best if you maintained Mailrise+ictoolkit as your own personal fork.

Regardless, happy to hear you found my code useful.

**Comment by IncognitoCoding** [comment]
I do not use the encryption piece in my environment, but I figured a few might find some use out of it. This pull request will link back to the release I included if anyone wanted to use it and request that I bring it back.

For now, I will clone a fresh instance from your main and refactor the HTML to text directly to Mailrise and send another pull request.

I will have the request sent here shortly.
